include/boost/random/splitmix64.hpp

```diff
@@ -0,0 +1,98 @@
+/* 
+ * Copyright Sebastiano Vigna 2015.
```

Comment:
Did Sebastiano Vigna write this code?

Reply:
The implementation of `next` is almost identical to his published C code. His code is all licensed under the Creative Commons license which is compatible with the Boost Software License.

Reply:
Just double checking this as creative commons isn't always BSL compatible, but there's an explicitly "public domain" version so this really is OK ;)